crates/jstz_cli/src/main.rs

```diff
@@ -21,30 +21,33 @@ use run::DEFAULT_GAS_LIMIT;
 use utils::AddressOrAlias;
 
 #[derive(Debug, Parser)]
-#[command(author, version)]
+#[command(name = "jstz", author = "TriliTech <contact@trili.tech>", version)]
 enum Command {
-    /// Commands related to the jstz sandbox.
+    /// 🏝️ Interact with the jstz sandbox.
```

Comment:
I think it would be a bit easier for user if instead of "Interact with", there was written "Start/Stop the local jstz sandbox". This would make it easier for the user to immediately understand what he can do with the sandbox without having to check `sandbox --help`.

Reply:
I'd prefer to avoid this, since these would bitrot quickly and aren't standard for nested commands

Reply:
Are emojis standard?